src/and.js

```diff
@@ -19,5 +23,5 @@ var _curry2 = require('./internal/_curry2');
  *      R.and(null, ''); => null
  */
 module.exports = _curry2(function and(a, b) {
-  return a && b;
+  return _hasMethod('and', a) ? a.and(b) : a && b;
```

Comment:
> && is not commutative for types with multiple "true" values:

good :point_right: 

so you're essentially treating `and` as an operator. The rationale was not immediately clear to me, but that makes perfect sense